packages/Tabs/src/Tabs.js

```diff
@@ -1,18 +1,21 @@
 import React from "react";
 import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
 import TabsContext from "./TabsContext";
 import Panel from "./components/Panel/Panel";
 import Panels from "./components/Panels/Panels";
 import Tab from "./components/Tab/Tab";
 import List from "./components/List/List";
 
 const propTypes = {
+  borderColor: PropTypes.string,
```

Comment:
How come we need this prop? Is a purple version needed sometimes? 

I would rather have something more controlled like `oneOf(['primary', 'secondary'])` so we maintain control / consistency.